release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1909,6 +1892,24 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 
 * Before this update, some of the sidebar items did not contain namespace context. Consequently, when links were opened from another browser, or opening links from a different active namespace, the web console does not switch to the correct namespace. With this update, the correct namespace is selected when opening the URL. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2039647[*BZ#2039647*])
 
+* Previously, when using the console to instantiate a template, its parameters get stored as a secret resource. When the template is removed, the secret stays. Consequently creating unnecessary buildup of secrets in the cluster. With this update, an ownership reference is added to the secret that maps to the template instance. Now, when the template instance is removed, the secret is also removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2015042[*BZ#2015042*])
+
+* With this update, the `jsonData` property has been deprecated and replaced with `data` in the `ping` source. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084438[*BZ#2084438*])
+
+* Previously, topology with more than 100 nodes was failing or lagging due to size. With this update, users are alerted to the lagging issue and offered the option to use `Search` to find a specific resource or continue with topology eliminating any lag. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060329[*BZ#2060329*])
+
+* Previously, The `8080` port was stalled on the service port list and random ports are deleted on a service with multiple ports. With this update, a condition in the `createService` function is specified to identify the current route `tartgetPort` and replace that prot from the service ports whenever a new `targetPort` is added in the edit application form. Now, when port 8080 is replaced as route port, it is removed from the service ports list and the new route ports list and the new route port gets added to the list. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2077943[*BZ#2077943*])
+
+* Previously, the git detection used to manage instance APIs to get repository information did not work for repositories from self hosted GitHub and Bitbucket. With this update, detection for self hosted GitHub and Bitbucket instance repositories works. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038244[*BZ#2038244*])
```

Comment:
You could place "git" in back-ticks if it refers to the `git` command. Otherwise, you could capitalise the word if it refers to the product name.